examples/localstack-spring-boot-sample/pom.xml

```diff
@@ -30,6 +30,14 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
```

Comment:
In Spring Boot application, configuring `<java.version>` property is sufficient to set Java version. We can remove the maven-compiler-plugin configuration.